README.rst

```diff
@@ -47,6 +47,13 @@ The S4-Clarity library lets developers interact with the Clarity API in fewer li
     if __name__ == "__main__":
         Normalization.main()
 
+Installation
+------------
+
+To install s4-clarity, simply use pip::
+
+    $ pip install s4-clarity
+
 Documentation
 -------------
-To build the documentation, run ``make html`` from the docs directory.
\ No newline at end of file
+To build the documentation, run ``make html`` from the docs directory.
```

Comment:
Who is the audience for this readme? It seems like the rest of the document is aimed at people just using the library from a pip install while this section is really aimed at someone downloading the repository and making changes.

Reply:
Good point. As running `make html` is the default Sphinx behaviour perhaps this should be removed.